internal/model/user.go

```diff
@@ -60,7 +60,7 @@ type PasswordForgot struct {
 type UpdateUser struct {
 	FirstName       string `bson:"first_name" json:"first_name"`
 	LastName        string `bson:"last_name" json:"last_name"`
-	Email           string `bson:"email" json:"email" validate:"required"`
+	Email           string `bson:"email" json:"email"`
```

Comment:
we have to validate the email
is there any reason why u removed this?

Reply:
Yes remember I was checking with email at first, now the Token is all that's needed